easybuild/easyconfigs/h/HISAT2/HISAT2-2.2.1-foss-2019b.eb

```
@@ -0,0 +1,54 @@
+##
+# This is a contribution from DeepThought HPC Service, Flinders University, Adelaide, Australia
+# Homepage: 	https://staff.flinders.edu.au/research/deep-thought
+#
+# Authors::	Robert Qiao <rob.qiao@flinders.edu.au>
+# License::	GPLv3.0
+#
+# Notes::
+# 2.2.1 - changes from Adam Huffman <adam.huffman@bdi.ox.ac.uk>
+##
+
+easyblock = 'MakeCp'
+
+name = 'HISAT2'
+version = '2.2.1'
+
+homepage = 'https://ccb.jhu.edu/software/%(namelower)s/index.shtml'
+description = """HISAT2 is a fast and sensitive alignment program for mapping next-generation sequencing reads
+ (both DNA and RNA) against the general human population (as well as against a single reference genome)."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+sources = [{
+    'source_urls': ['https://cloud.biohpc.swmed.edu/index.php/s/fE9QCsX3NH4QwBi'],
+    'download_filename': 'download',
+    'filename': '%(namelower)s-%(version)s-source.zip',
+}]
+checksums = ['48e933330d4d8470d2b3dfe7ec3918f2e98a75f7381891e23b7df1fb4f135eb1']
+
+dependencies = [
+    ('NGS', '2.10.4', '-Java-11'),
+    ('ncbi-vdb', '2.10.4'),
+]
+
+buildopts = 'CC="$CC" CPP="$CXX" RELEASE_FLAGS="$CFLAGS" '
+buildopts += "USE_SRA=1 NCBI_NGS_DIR=$EBROOTNGS NCBI_VDB_DIR=$EBROOTNCBIMINVDB"
+
+local_executables = ['hisat2', 'hisat2-align-l', 'hisat2-align-s', 'hisat2-build', 'hisat2-build-l', 'hisat2-build-s',
+                     'hisat2-inspect', 'hisat2-inspect-s', 'hisat2-inspect-l', 'hisat2-repeat', 'extract_exons.py',
+                     'extract_splice_sites.py', 'hisat2_extract_exons.py', 'hisat2_extract_snps_haplotypes_UCSC.py',
+                     'hisat2_extract_snps_haplotypes_VCF.py', 'hisat2_extract_splice_sites.py',
+                     'hisat2_read_statistics.py', 'hisat2_simulate_reads.py']
+files_to_copy = [
+    (local_executables, 'bin'),
+    ('scripts'),
+    ('example'),
+]
```

@verdurin the parentheses are not needed if you copy to the root installdir, so you can just do:
```suggestion
files_to_copy = [(local_executables, 'bin'), 'scripts', 'example']
```

---

@smoors done.